src/main/java/workers/ICSExporter.java

```diff
@@ -27,30 +23,49 @@
 import util.InvalidSessionException;
 
 /** Use Case class for exporting a Schedule into an ICS file */
-public class ScheduleExporter {
+public class ICSExporter extends Exporter {
 
-    private static int numFiles = 0;
+    private int numFiles = 0;
```

Comment:
this is already in the exporter class but its called num, either name works so you can change one or the other